vNext/test/NonSiloTests/Orleans.NonSiloTests.csproj

```diff
@@ -79,29 +111,103 @@
+      <Link>Config_TestSiloConfig.xml</Link>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+    <Content Include="..\..\..\test\TesterInternal\DevTestClientConfiguration.xml">
+      <Link>DevTestClientConfiguration.xml</Link>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+    <Content Include="..\..\..\test\TesterInternal\DevTestServerConfiguration.xml">
+      <Link>DevTestServerConfiguration.xml</Link>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+    <Content Include="..\..\..\test\Tester\DeploymentTests\Configuration\TestAbsolutePaths\OrleansConfiguration.xml">
+      <Link>OrleansConfiguration.xml</Link>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+    <Content Include="..\..\..\test\Tester\SerializationTests\ConfigurationTests\ClientConfigurationForSerializer2.xml">
+      <Link>SerializationTests\ConfigurationTests\ClientConfigurationForSerializer2.xml</Link>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
     <Content Include="..\..\..\test\Tester\SerializationTests\ConfigurationTests\ClientConfigurationForSerializer6.xml">
       <Link>SerializationTests\ConfigurationTests\ClientConfigurationForSerializer6.xml</Link>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
     <Content Include="..\..\..\test\Tester\SerializationTests\ConfigurationTests\ClientConfigurationForSerializer7.xml">
       <Link>SerializationTests\ConfigurationTests\ClientConfigurationForSerializer7.xml</Link>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
     <Content Include="..\..\..\test\Tester\SerializationTests\ConfigurationTests\ClientConfigurationForSerializer8.xml">
       <Link>SerializationTests\ConfigurationTests\ClientConfigurationForSerializer8.xml</Link>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+    <Content Include="Config_LogConsumers-ClientConfiguration.xml">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+    <Content Include="Config_LogConsumers-OrleansConfiguration.xml">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+    <Content Include="SerializationTests\ConfigurationTests\ClientConfigurationForSerializer.xml">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+    <Content Include="SerializationTests\ConfigurationTests\ClientConfigurationForSerializer3.xml">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+    <Content Include="SerializationTests\ConfigurationTests\ClientConfigurationForSerializer4.xml">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+    <Content Include="SerializationTests\ConfigurationTests\ClientConfigurationForSerializer5.xml">
```

Comment:
I copied over the files, instead of link them, because it need a slightly different config in vNext. See below